apps/frontend/src/app/pages/dashboard/dashboard.component.scss

```diff
@@ -26,24 +27,3 @@
     }
   }
 }
-
-::ng-deep body {
-  &::before {
-    content: '';
-    position: fixed;
-    left: 0;
-    right: 0;
-    bottom: 0;
-    height: 100vh;
-    z-index: -1;
-    will-change: transform;
-    min-height: 100vh;
-    min-width: 100vw;
-    background: url('/assets/images/splash.webp');
```

Comment:
Is this image still used somewhere? Should it be deleted?

Reply:
`not-found-main` uses it on the frontpage. I'd rather alter the frontpage as little as possible until we're ready to completely delete it, I can make a note in https://github.com/momentum-mod/website/issues/739 to remove it then.

Reply:
Perfect thanks